src/Development/IDE/Core/Rules.hs

```diff
@@ -342,22 +347,27 @@ getSpanInfoRule :: Rules ()
 getSpanInfoRule =
     define $ \GetSpanInfo file -> do
         tc <- use_ TypeCheck file
+        packageState <- hscEnv <$> use_ GhcSession file
         deps <- maybe (TransitiveDependencies [] [] []) fst <$> useWithStale GetDependencies file
         let tdeps = transitiveModuleDeps deps
+#ifdef GHC_LIB
         parsedDeps <- uses_ GetParsedModule tdeps
+#else
+        let parsedDeps = []
```

Comment:
How does this work in the GHC_LIB case? I don't understand why the CPP is here at all actually.

Reply:
The need for this comes from `Development.IDE.Spans.Documentation`. I'll add some comments